src/general/GameProperties.cs

```diff
@@ -110,6 +110,13 @@ public static GameProperties StartNewEarlyMulticellularGame(WorldGenerationSetti
 
         game.EnterPrototypes();
 
+        if (freebuild)
+        {
+            game.EnterFreeBuild();
+            game.GameWorld.GenerateRandomSpeciesForFreeBuild();
```

Comment:
Thanks for contributing to Thrive. Someone will do a full review once the diff for MainMenu.cs is fixed (it seems you converted all spaces to tabs), but I'll just quickly comment that could you place a todo comment here like: `// TODO: generate some other multicellular species for the player to interact with`.